packages/ui/src/views/MainContent.vue

```diff
@@ -139,8 +110,7 @@ export default {
     })
 
     const type = computed(() =>
-      player.incubating ||
-      (player.creatureData && parseInt(player.creatureData.tokenId) !== 0)
+      player.incubating || (player.data && parseInt(player.data.tokenId) < 0)
```

Comment:
We don't have `incubating` anymore. Could you update it or add comment to update it when we have the name from the contract?